atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1010,6 +1010,9 @@ void WebContents::TabDetachedAt(content::WebContents* contents, int index) {
 
   if (owner_window() && owner_window()->browser())
     owner_window()->browser()->tab_strip_model()->RemoveObserver(this);
+
+  auto api_web_contents = CreateFrom(isolate(), contents);
```

Comment:
you don't need this because the 1008-1009 filter out detached notifications for other atom_api_web_contents